code/__DEFINES/is_helpers.dm

```diff
@@ -4,6 +4,21 @@
 
 #define in_range(source, user) (get_dist(source, user) <= 1)
 
+//Turfs
```

Comment:
yeah, I only did partial ports for some things just to get the gas interactions working